src/scripts/main.js

```diff
@@ -1,7 +1,106 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
 
-// Write your code here
+const game = new Game();
+
+const startButton = document.querySelector('.button-start');
+const scoreElement = document.querySelector('.game-score');
+const messageStart = document.querySelector('.message-start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+
+function updateBoard() {
+  const state = game.getState();
+
+  state.forEach((row, rowIndex) => {
+    row.forEach((value, colIndex) => {
+      const cell = document.querySelector(
+        `[data-position="${rowIndex}-${colIndex}"]`,
+      );
+
+      if (!cell) {
+        return;
+      }
+
+      cell.textContent = value !== 0 ? value : '';
+      updateCellClass(cell, value);
+    });
+  });
+}
+
+function updateCellClass(cell, value) {
+  cell.className = 'field-cell';
+
+  if (value !== 0) {
+    cell.classList.add(`field-cell--${value}`);
+  }
+}
+
+function updateScore() {
+  scoreElement.textContent = game.getScore();
+}
+
+function updateMessages() {
+  const statusGame = game.getStatus();
+
+  messageStart.classList.add('hidden');
+  messageLose.classList.add('hidden');
+  messageWin.classList.add('hidden');
+
+  switch (statusGame) {
+    case 'win':
+      messageWin.classList.remove('hidden');
+      break;
+    case 'lose':
+      messageLose.classList.remove('hidden');
+      break;
+    case 'idle':
+      messageStart.classList.remove('hidden');
+      break;
+    default:
+      break;
+  }
+}
+
+startButton.addEventListener('click', () => {
+  if (startButton.classList.contains('start')) {
+    game.start();
```

Comment:
Ensure that the `start` method in the `Game` class correctly initializes the game state. There was a typo in the `Game` class regarding the method name `addRandomTitle`, which should be `addRandomTile`. Make sure this is corrected in the `Game` class to avoid runtime errors.